internal/controller/ols_app_server_assets_test.go

```diff
@@ -1908,18 +1908,18 @@ func generateRandomSecret() (*corev1.Secret, error) {
 	return &secret, nil
 }
 
-func generateRandomConfigMap() (*corev1.ConfigMap, error) {
-	configMap := corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "test-configmap",
-			Namespace: "openshift-lightspeed",
-		},
-		Data: map[string]string{
-			"service-ca.crt": "random ca cert content",
-		},
-	}
-	return &configMap, nil
-}
+//func generateRandomConfigMap() (*corev1.ConfigMap, error) {
+//	configMap := corev1.ConfigMap{
+//		ObjectMeta: metav1.ObjectMeta{
+//			Name:      "test-configmap",
+//			Namespace: "openshift-lightspeed",
+//		},
+//		Data: map[string]string{
+//			"service-ca.crt": "random ca cert content",
+//		},
+//	}
+//	return &configMap, nil
+//}
```

Comment:
@blublinsky we can remove this function ?

Reply:
I think we should
It was there and I did not want to delete it straight,cause I was not the one putting it in